packages/state-transition/src/util/epochShuffling.ts

```diff
@@ -109,9 +110,46 @@ export function computeEpochShuffling(
 ): EpochShuffling {
   const activeValidatorCount = activeIndices.length;
 
-  const shuffling = activeIndices.slice();
   const seed = getSeed(state, epoch, DOMAIN_BEACON_ATTESTER);
-  unshuffleList(shuffling, seed);
+  const shuffling = unshuffleList(activeIndices, seed, SHUFFLE_ROUND_COUNT);
+
+  const committeesPerSlot = computeCommitteeCount(activeValidatorCount);
+
+  const committeeCount = committeesPerSlot * SLOTS_PER_EPOCH;
+
+  const committees: Uint32Array[][] = [];
```

Comment:
preallocate with an array of `SLOTS_PER_EPOCH` then set each index instead of `push`

Reply:
Ok. Updated.